Commander origin (#484)
MewsSystems/gitbook-connector-api: operations/reservations.md

| `EndUtc` | string | required | Reservation end in UTC timezone in ISO 8601 format. |
| `Number` | string | required | Confirmation number of the reservation in Mews. |
| `State` | string [Service order state](#service-order-state) | required | State of the reservation. |
| `Origin` | string [Service order origin](productserviceorders.md#service-order-origin) | required | Origin of the reservation. |

> | `CommanderOrigin` | string [Commander origin](#commander-origin) | optional | Specification of origin in case of Origin is `Commander`. |

Reviewer: ```suggestion
| `CommanderOrigin` | string [Commander origin](#commander-origin) | optional | Further detail about origin in case of Origin `Commander`. |
```

    ]
}
```

> #### Commander origin

Reviewer 2: suggestion: I think we should explain here the meaning of commander origin and introduce the list as incomplete example or mention that more values may be added in the future, so the developers don't attempt to do strict validation on the value.

```suggestion
#### Commander origin

In case the reservation is created with Mews Operations, `CommanderOrigin` specifies how the reservation was created, for example:
```
```

Author: Since this is just definition of enum, you should not put explanation here since you don't know where it could be linked in the future. Explenation is done on level of endpoint 👍

And that enum can change in the future is done via `...` in the end
